packages/react-native-codegen/src/parsers/__tests__/parsers-primitives-test.js

```diff
@@ -351,7 +351,20 @@ describe('emitPromise', () => {
     it('throws an IncorrectlyParameterizedGenericParserError error', () => {
       const nullable = false;
       expect(() =>
-        emitPromise(moduleName, typeAnnotation, parser, nullable),
+        emitPromise(
+          moduleName,
+          typeAnnotation,
+          parser,
+          nullable,
+          // mock translateTypeAnnotation function
+          {},
+          {},
+          function <T>(_: () => T) {
+            return null;
+          },
+          false,
+          (_, elementType) => elementType,
```

Comment:
Please consider hoisting these repeated arguments to the top of the describe method. This will make the change easier to follow, as people will know what the empty object means for `emitPromise`.

Reply:
Fixed